src/redux/books/categorySlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    checkStatus: (state) => state.status === 'Under construction',
+  },
```

Comment:
- [ ] Please kindly ensure that according to the project requirement,  this reducer should check the status and always returns "Under construction". 

<img width="302" alt="Untitled" src="https://github.com/kalbek/bookstore/assets/103386183/a3488c6d-80af-44a0-9ab8-deacc9b3aa5e">